src/core/SpineGenerator.ts

```diff
@@ -101,23 +89,22 @@ export class MeshGenerator {
     }
     this._vertexCount = vertexCount;
     this._prepareBufferData(this._vertexCount);
-    const { _spineMesh } = this;
-    _spineMesh.initialize(this._engine, this._vertexCount);
-    meshRenderer.mesh = _spineMesh.mesh;
+    const { _spinePrimitive } = this;
+    _spinePrimitive.initialize(this._engine, this._vertexCount);
+    this._renderer._primitive = _spinePrimitive.primitive;
   }
 
-  buildMesh(skeleton: Skeleton) {
+  buildPrimitive(skeleton: Skeleton) {
```

Comment:
`const { useClipping = true, zSpacing = 0.01 } = this._setting || {};` new Object every frame?